frontend/src/pages/spotnet/dashboard/Dashboard.jsx

```diff
@@ -116,52 +122,84 @@ const Dashboard = ({ walletId }) => {
+        <div className="bg-custom-health d-flex align-items-center px-4 py-3 rounded bg-card-health">
+          <span className="dashboard-text-color health-text-size">Health factor:</span>
+          <span className="text-white text-style">{healthFactor}</span>
         </div>
-
-        <h1 className="text-white text-center zkLend-text">zkLend Position</h1>
-
-        <div className="health-factor-wrapper">
-          <div className="card card-health-factor mx-auto d-flex flex-column align-items-center justify-content-center card-shadow">
-            <div className="bg-custom-health d-flex align-items-center rounded bg-card-health">
-              <span className="dashboard-text-color health-text-size">Health factor:</span>
-              <span className="text-white text-style">{healthFactor}</span>
+      </div>
+      <div className="mb-4 d-flex flex-row justify-content-center cards-custom">
+        {cardData.map((card, index) => (
+          <div key={index} className="card card-custom-styles d-flex flex-column align-item-center card-shadow">
+            <header className="card-header bg-custom-color text-light text-center card-shadow">
+              <div className="d-flex align-items-center justify-content-center">
+                <card.icon className="card-icons rounded-circle" />
+                <h1
+                  className="ms-2 icon-text-gap mb-0 text-style"
+                  style={{
+                    color: card.title === 'Borrow' ? '#FF5A5F' : '#4CAF50', // Direct color values
+                  }}
+                >
+                  {card.title}
+                </h1>
+              </div>
+            </header>
+            <div className="card-body card-body-custom">
+              <div className="d-flex flex-column align-items-center bg-custom-color rounded">
+                <div className="d-flex align-items-center mb-3">
+                  <card.currencyIcon className="card-icons rounded-circle" />
+                  <span className="ms-2 icon-text-gap text-style">{card.currencyName}</span>
+                </div>
+                <div className="d-flex align-items-center">
+                  <span className="dashboard-text-color balance-text-size">Balance:</span>
+                  <span
+                    className="ms-2 borr-text icon-text-gap text-style"
+                    style={{
+                      color: card.title === 'Borrow' ? '#FF5A5F' : '#4CAF50', // Direct color values
+                    }}
+                  >
+                    {card.balance}
+                  </span>
+                </div>
+                {card.title === 'Collateral & Earnings' && (
+                  <div className="sum-info">
+                    <span className="start-sum">Start sum: {startSum} $</span>
+                    <span className="current-sum">
+                      Current sum: <span style={{ color: currentSum < startSum ? '#FF5A5F' : '#4CAF50' }}>{currentSum} $</span>
```

Comment:
it should be different css classnames on condition (just like you had before), and in css you should use CSS variables for colors